### DISCUSSION TITLE
[Feature] sort album by upload date

### DISCUSSION THREAD

**Discussion by sebbi08** [immich-app/immich#7375] [Feature Request]
### The feature

Let's say you have shared an album with your family, and they also upload images to it.

Now some of them might be late with uploading and someone else already downloaded all the images and now wants to only redownload the newest images.

That's where a sort by upload date would be nice

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#1689
